Convex_hull_3/include/CGAL/Convex_hull_3/distance.h

```diff
@@ -0,0 +1,506 @@
+* @tparam NamedParameters_2 a sequence of \ref bgl_namedparameters "Named Parameters"
+*
+* @param ch1 the first convex hull
+* @param ch2 the second convex hull
+* @param np1 an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+* @param np2 an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+*
+* \cgalNamedParamsBegin
+*   \cgalParamNBegin{vertex_point_map}
+*     \cgalParamDescription{a property map associating points to the vertices of `ch1` (`ch2`)}
+*     \cgalParamType{a model of `ReadablePropertyMap` whose value types are the same for `np1` and `np2`}
+*     \cgalParamDefault{boost::get(CGAL::vertex_point, g)}
+*     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t` must be available in `IncidenceGraph`.}
+*   \cgalParamNEnd
+*   \cgalParamNBegin{geom_traits}
+*     \cgalParamDescription{An instance of a geometric traits class}
+*     \cgalParamType{a class model of `Kernel`}
+*     \cgalParamDefault{a \cgal kernel deduced from the point type, using `CGAL::Kernel_traits`}
+*     \cgalParamExtra{`np1` only}
+*   \cgalParamNEnd
+*   \cgalParamNBegin{number_of_iterations}
+*     \cgalParamDescription{if not `0` (no limit), indicates the maximum number of iterations performed by the algorithm.
+                            If this value is not `0`, then the return value can be zero even if the convex hulls does not intersect.
+*                           However, the value reported remains a lower bound of the distance between the convex.}
+*     \cgalParamType{a positive integer convertible to `std::size_t`}
+*     \cgalParamExtra{`np1` only}
+*     \cgalParamDefault{`0`}
+*   \cgalParamNEnd
+* \cgalNamedParamsEnd
+*
+*/
+template <class PolygonMesh,
+          class NamedParameters_1 = parameters::Default_named_parameters,
+          class NamedParameters_2 = parameters::Default_named_parameters>
+FT separation_distance(const Convex_hull_hierarchy<PolygonMesh>& ch1, const Convex_hull_hierarchy<PolygonMesh>& ch2,
+                       const NamedParameters_1& np1 = parameters::default_values(),
+                       const NamedParameters_2& np2 = parameters::default_values());
+
+#else
+
+template <class Convex1, class Convex2,
+          class NamedParameters_1 = parameters::Default_named_parameters,
+          class NamedParameters_2 = parameters::Default_named_parameters>
+typename predicates_impl::GetGeomTraitsFromConvex<Convex1, NamedParameters_1>::type::FT
+separation_distance(const Convex1& c1, const Convex2& c2,
+                    const NamedParameters_1& np1 = parameters::default_values(),
+                    const NamedParameters_2& np2 = parameters::default_values()){
+  using CGAL::parameters::choose_parameter;
+  using CGAL::parameters::get_parameter;
+
+  //The function need exact computation to works correctly
```

Comment:
This should be indicated clearly somewhere. What happen if we are using EPICK? Since it is a lower bound, can't we use intervals?